contracts/cw20-escrow/src/error.rs

```diff
@@ -0,0 +1,26 @@
+use cosmwasm_std::StdError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
```

Comment:
I like this one

Reply:
I think I got those from `cosmwasm-examples`.